package-overrides/github/a5hik/ng-sortable@1.1.9.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "css": "*"
+  },
+  "directories": {
+    "lib": "dist"
+  },
+  "main": "ng-sortable",
+  "registry": "jspm",
+  "shim": {
+    "ng-sortable": {
+      "deps": [
+        "./ng-sortable.min.css!"
```

Comment:
Might be better to change the [main parsing](https://github.com/jspm/jspm-cli/blob/0.10.6/lib/package.js#L659) to always trust the file extension

So 
```js
"main": "ng-sortable"
// >>> loads "ng-sortable" without extension
"main": "ng-sortable.js"
// >>> loads "ng-sortable.js"
"main": "ng-sortable.css"
// >>> loads "ng-sortable.css" with the css plugin
```
 What you think @guybedford